packages/loopring_v2/contracts/helper/OrderHelper.sol

```diff
@@ -423,6 +413,7 @@ library OrderHelper {
     {
         if (!tokenSpendable.initialized) {
```

Comment:
Make sure the spendables are correctly set. There could be some strange combinations (order with/without a broker but with the same owner) but probably not really a problem in practice because that won't happen.